core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -16,21 +16,19 @@
  * @fileoverview Data and controllers for the Oppia 'edit preferences' page.
  */
 
-import Cropper from 'cropperjs';
-
 require('cropperjs/dist/cropper.min.css');
```

Comment:
Remove this too?

Reply:
Yes, because the inline modal was using Cropper and not the file itself. I've separated it in a new file and I'm importing Cropper in this new file.